docs/api/FluxMixin.md

```diff
@@ -40,6 +40,17 @@ fluxMixin({
 });
 ```
 
+In some situations the data retrieved from one store depends on the state from another, you can use an array of store keys with a custom state getter to ensure the components state is updated when either store changes:
+
+```js
+fluxMixin(
+  ['posts', 'session'],
+  ([postStore, sessionStore]) => ({
```

Comment:
is an inline comment in the example a good spot for that?

could be a good opportunity mention why its an array instead of separate arguments.